cluster-configuration/services-configuration.yaml

```diff
@@ -34,27 +34,27 @@ cluster:
   docker-registry-info:
 
     # If public, please fill it the same as your username
-    docker-namespace: your_registry_namespace
+    docker-namespace: openpai
 
     # E.g., gcr.io. If public，fill docker_registry_domain with word "public"
     # docker_registry_domain: public
-    docker-registry-domain: your_registry_domain
+    docker-registry-domain: docker.io
     # If the docker registry doesn't require authentication, please leave docker_username and docker_password empty
-    docker-username: your_registry_username
-    docker-password: your_registry_password
+    docker-username: 
```

Comment:
Is it grammatically valid (nothing behind the colon)? I tested comment these two fields out, same effect.

Reply:
do you leaving these empty will introduce a bug?

---
In reply to: [195366241](https://github.com/Microsoft/pai/pull/727#discussion_r195366241) [](ancestors = 195366241)